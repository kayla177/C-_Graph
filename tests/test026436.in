ENTITY T6FV3BHH name_T6FV institution
ENTITY CNL3N9EI name_CNL3 paper
ENTITY 2HPHMPJS name_2HPH journal
RELATIONSHIP T6FV3BHH presented_at CNL3N9EI 0.7
PRINT T6FV3BHH
HIGHEST
EXIT
