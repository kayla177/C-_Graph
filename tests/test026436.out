success
success
success
success
CNL3N9EI
T6FV3BHH CNL3N9EI 0.7
